canonical-web-and-design/ubuntu.com
# Rebuild /iot/gateways page

## Done

- Rebuild /internet-of-things/gateways page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/internet-of-things/gateways](http://0.0.0.0:8001/internet-of-things/gateways)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy doc](https://docs.google.com/document/d/18ffTcNolrtJS9AvjMfa8KW94uLfC0KzmHtTe-jMYmlo)


## Issue / Card

Fixes #5655
